app/assets/stylesheets/refills/_header.scss

```diff
@@ -0,0 +1,87 @@
+.header-2 {
+  $base-spacing: 1.5em !default;
+  $base-accent-color: $bookshelf-red-2 !default;
+  $medium-screen: em(640) !default;
+  $large-screen: em(860) !default;
+
+  ul {
+    margin: 0;
+    padding: 0;
+    line-height: 1.5em;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  $header-background: $base-accent-color;
+  $header-color: white;
+  $header-link-color: transparentize($header-color, 0.2);
+  $header-disclaimer-color: transparentize($header-color, 0.6);
+
+  background: $header-background;
+  padding: $base-spacing;
+  width: 100%;
+  display: inline-block;
+
+  .header-logo {
+    color: $white;
+    font-weight: bold;
+    margin-right: 1em;
+    margin-bottom: 1em;
+
+    @include media($large-screen) {
+      float: left;
+      margin-bottom: 0;
+    }
+  }
+
+  .header-logo img {
+    height: 1.6em;
+  }
+
+  ul {
+    margin-bottom: 1em;
+
+    @include media($large-screen) {
+      float: left;
+      line-height: 1.8em;
+      margin-left: 1em;
+      margin-bottom: 0;
+    }
+  }
+
+  ul li {
+    font-weight: 800;
+    padding-right: 1em;
+
+    @include media($large-screen) {
+      display: inline;
+      text-align: left;
+    }
+  }
+
+  ul li a {
```

Comment:
Merge rule `ul li a` with rule on line 7<br>Selector should have depth of applicability no greater than 2, but was 4